data/map systems.txt

```diff
@@ -5984,6 +6157,67 @@ system Atik
 		distance 1905.05
 		period 270.619
 
+system Atlas
```

Comment:
```suggestion
system Algedi
```
This breaks the alphabetical thing we got in this file, sorry – ~~this would be easier in a PR to your branch~~

Reply:
I can reorder things later - it's this way so that other people can discuss the specifics _here_